tests/integration/awslambda/test_lambda_api.py

```diff
@@ -2242,6 +2353,327 @@ def test_add_lambda_permission_aws(
         get_policy_result = lambda_client.get_policy(FunctionName=function_name)
         snapshot.match("get_policy", get_policy_result)
 
+    @pytest.mark.skipif(condition=is_old_provider(), reason="not supported")
+    @pytest.mark.skip(reason="wip not yet implemented")
+    # TODO: validate + set marker
+    # TODO: Generalize and minimize test for all lambda revision ids
+    # @pytest.mark.aws_validated
+    def test_lambda_permission_revisions(
+        self, lambda_client, iam_client, create_lambda_function, account_id, snapshot
+    ):
+        """Tests the behavior of revision id for lambda permissions."""
+        # TODO: double-check whether it's really just the revision id of the lambda function and not a separate one!!!
```

Comment:
is this still relevant? If I saw correctly this isn't the case with the current implementation anyway :thinking:

Reply:
removed it for now.
let's move that to a follow-up PR addressing revisions for lambda in general